src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const populationArray = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+const countriesCount = populationArray.length;
+let total = 0;
+
+populationArray.forEach((stringPopulation) => {
+  total += parseInt(stringPopulation.textContent.replace(/,/g, ''), 10);
+});
+
+const average = Math.round(total / countriesCount);
```

Comment:
Be cautious of division by zero. If `countriesCount` is zero, this line will result in a division by zero error. Consider adding a check to ensure `countriesCount` is greater than zero before performing the division.